buildSrc/src/main/minimumRuntime/org/elasticsearch/gradle/LoggedExec.java

```diff
@@ -63,7 +63,10 @@ public void setSpoolOutput(boolean spoolOutput) {
             out = new LazyFileOutputStream(spoolFile);
             outputLogger = logger -> {
                 try {
-                    Files.lines(spoolFile.toPath()).forEach(logger::error);
+                    // the file may not exist if the command never output anything
+                    if (Files.exists(spoolFile.toPath())) {
```

Comment:
Is there any real benefit in doing `Files.exists(spoolFile.toPath)` vs simply `spoolFile.exists()`?